test/run-drun/upgrade-migration.drun

```diff
@@ -0,0 +1,13 @@
+# SKIP ic-ref-run
```

Comment:
I believe it would be good to also test all these with graph copy.

Reply:
Indeed. It seems to be crashing...

Reply:
Now fixed and tested